src/pages/home/_ticker.js

```diff
@@ -189,21 +44,66 @@ const getTickerMarkets = active_symbols => {
         synthetic = shuffle(synthetic).slice(0, volatility_count)
     }
     if (forex.length) forex = shuffle(forex).slice(0, forex_count)
-
-    return [...synthetic, ...forex]
+    ;[...synthetic, ...forex].forEach(symbol =>
```

Comment:
still has a `;` here that shouldn't be here, if it's from prettier you can add a prettier ignore to this line else put it in a variable

Reply:
done